tests/api_test/api_test.cpp

```diff
@@ -1334,3 +1334,44 @@ TEST_CASE("test_ringbuffer_wraparound", "[stress]")
     // Clean up.
     bpf_object__close(object);
 }
+
+TEST_CASE("Test program order", "[native_tests]")
```

Comment:
I don't understand how this tests program order.  It just tests that "program1" returns 1, "program2" returns 2, etc., independent of what order they appear in.  Can you elaborate on this?

Reply:
Added comment to describe how the test validates the order.